src/WebJobs.Extensions.CosmosDB/Config/CosmosDBExtensionConfigProvider.cs

```diff
@@ -124,18 +124,18 @@ internal string ResolveConnectionString(string attributeConnectionString)
             return _options.ConnectionString;
         }
 
-        internal ICosmosDBService GetService(string connectionString, string preferredLocations = "", bool useMultipleWriteLocations = false)
+        internal ICosmosDBService GetService(string connectionString, string preferredLocations = "", bool useMultipleWriteLocations = false, bool useDefaultJsonSerialization = false)
         {
             string cacheKey = BuildCacheKey(connectionString, preferredLocations, useMultipleWriteLocations);
```

Comment:
You are missing here the new parameter. Cache Key should have all the parameter values

Reply:
done

Reply:
Should I mark them as resolved or should I wait for the reviewer to resolve?  My office switched to the reviewer as people where doing half or less changes and marking it resolved.